src/components/StockList.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import '../css/StockList.css';
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import SearchStock from './SearchStock';
+import amex from '../assets/amex.png';
+import nasdaq from '../assets/Nasdaq-Logo.svg';
+import nyse from '../assets/NYSE-logo.svg';
+
+const StockList = () => {
+  let { stocks } = useSelector((store) => store.stocks);
+  const { isLoading } = useSelector((store) => store.stocks);
+  const { searchTerm } = useSelector((store) => store.stocks);
+
+  if (isLoading) {
+    return (
+      <div>
+        Loading...please wait
+      </div>
+    );
+  }
+
+  const filteredStocks = stocks.filter(
+    (stock) => stock.companyName.toLowerCase().includes(searchTerm.toLowerCase()),
+  );
+
+  stocks = filteredStocks;
+
+  return (
+    <div>
+      <div className="header-container">
+        <div className="logo-container">
+          <img src={nasdaq} alt="nasdaq" />
+          <img src={nyse} alt="nyse" />
+          <img src={amex} alt="amex" />
+        </div>
+        <div className="title-container">
+          <p className="page-title">US capital market</p>
+        </div>
+      </div>
+      <div className="mid-section">
+        <p className="stats-text">Stats By Stocks</p>
+        <SearchStock />
+      </div>
```

Comment:
- [ ] 👍 Great work on your project! I noticed a minor alignment issue with the "Stats By Stocks" text. In the design, this text should be horizontally aligned with the navbar's "arrow-left" icon. To achieve this, I recommend adjusting the positioning of this text to match the design, ensuring a perfect layout. 

- [ ] Second,  consider adjusting the height of the input field to match the design's specified height for the "mid-section." Additionally, align the input component to the right to enhance the visual alignment with the design.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/AliRezaBmeDu/metrics-webapp/assets/55840999/cbace764-e701-4047-a257-3a25ec37e69a)| ![image](https://github.com/AliRezaBmeDu/metrics-webapp/assets/55840999/31d250b3-a750-48b9-9aa6-88720e976eaa)|

Your attention to these details will help create a polished and visually consistent user interface. Keep up the good work! 🚀👨‍💻🎨

Reply:
done